### ISSUE TITLE
@error is not recognized even if it's declared in at-rule-no-unknown[ignoreAtRules]

### ISSUE THREAD

**Issue by hinok** [shinnn/vscode-stylelint#60]
Given scss code

```scss
@mixin test {
  @error 'Test';
}
```

`vscode-stylelint` reports as invalid displaying errors:

```
[scss] } expected
[scss] at-rule or selector expected
```

When I run stylelint cli

```bash
> stylelint src/**/*.scss --syntax scss
```

there are no errors, code is valid and this is desired result.

**.stylelintrc**

```json
{
  "extends": "stylelint-config-standard",
  "rules": {
    "at-rule-empty-line-before": ["always", {
      "except": [
        "blockless-after-blockless",
        "first-nested",
      ],
      "ignore": ["after-comment"],
    }],
    "selector-pseudo-class-no-unknown": [true, {
      "ignorePseudoClasses": [
        "export",
        "import",
        "global",
        "local",
      ]
    }],
    "property-no-unknown": [true, {
      "ignoreProperties": [
        "composes",
      ]
    }],
    "at-rule-no-unknown": [true, {
      "ignoreAtRules": [
        "value",
        "extend",
        "at-root",
        "debug",
        "error",
        "warn",
        "if",
        "else",
        "for",
        "each",
        "while",
        "mixin",
        "include",
        "content",
        "return",
        "function"
      ]
    }],
    "declaration-empty-line-before": ["never"]
  }
}
```

**Comment by shinnn** [comment]
The warning starting with `[scss] ` prefix is not from this extension, but from the VSCode built-in CSS linter.

**Comment by hinok** [comment]
Aaaaaaghhhh, my fault, sorry for that.

**Comment by shinnn** [comment]
@hinok No problem!

**Comment by eschaefer** [comment]
My solution to this was disabling the built-in VSCode scss checker in User Settings:
```
"scss.validate": false,
"stylelint.enable": true,
```
